0. Kirish/0. Kirish.md

```diff
@@ -11,3 +11,11 @@ Ushbu Me'yorning qiymati sanoatda endi yo'lini topayotganlar yoki yo'l topishda
 Men ushbu Me'yorda ko'plab me'yorlarni jamladim, so‘ng mualliflik qildim. Umid qilamanki, bu ko'p chalkashliklarni bartaraf qiladi va muhandislarga eng muhim masalalarga e'tibor qaratish imkonini beradi - texnologiyadan yuqori maqsadlar yo'lida foydalanish va unga teng qiymatli maqsadlarni belgilashga ko'maklashadi. Dasturiy ta'minotni loyihalash san'ati va ilmi uzoq yo'lni bosib o'tdi va bugungi kunda inson ega bo'lishi mumkin bo'lgan eng kuchli vositalardan biri ekanligini isbotladi. U to'g'ri tanishtiruvga munosib. Va biz yoshlarga u haqida qanday ta'lim berishimiz muhim.
 
 Mohiyatan, Me'yor - chinakam mustahkam tizimlarga erishishimiz uchun dizayn va ishlanmalarimizni boyitishda davom etuvchi SOLID tamoyillari va boshqa ko'plab amaliyot va namunalarning men tomonimdan taqdim etilayotgan talqinidir. Me'yor har bir muhandisga kundalik ishlarida yo'l-yo'riq topishida yordam berishga qaratilgan. Ammo yanada muhimi, Me'yor har bir muhandisga oyga qo'nadigan, eng murakkab muammolarni hal qiladigan va insoniyatning omon qolishi va evolyutsiyasini ta'minlaydigan mustahkam tizimlarni qurishi kerak bo'lgan paytda, ular kerakli ko'rsatmalarga ega bo'lishini ta'minlaydi.
+
+Me'yor agnostik-texnologiyaga mo'ljallangan. Uning printsiplari har qanday dasturlash tiliga taalluqli bo'lishi mumkun va uning uch xil asosi, dasturiy ta'minotdan tashqari har qanday ishlab chiqish yoki dizayn qarorlarini boshqarishi mumkin. Men C# tili va .NET frameworkdan faqat ushbu standart tushunchalarini amalga oshirish uchun foydalanaman. Ammo shuni bilingki, ushbu standartni shakllantirishning dastlabki bosqichlarida men Scala-dan dasturlash tili sifatida ko'p foydalanardim. Me'yor biron bir texnologiyaga bog'liq bo'lmasligi kerak va tanlangan tildan qat'iy nazar, unga amal qilmoqchi bo'lganlar uchun cheklov bo'lmasligi kerak. Shuningdek bu, muhandislar avlodlarining ergashishlari, takomillashtirishlari yoki o'zlari ishlab chiqishlari uchun ilhom rolini o'ynashga mo'ljallangan. Shu bilan bir qatorda dasturiy ta'minotni me'yorlarsiz yaratish mumkun, lekin eng yaxshi vaqtni eng yaxshi harakatlarga sarflashga kelganda tartibsizlik va adolatsizlikka duchor bo'ladi. Bugungi kunda sanoatimiz me'yorlashtirish nuqtai nazaridan betartiblikda. Malakasiz shaxslar etakchilik lavozimlariga ega bo'lishi va baxtsiz qarorlar qabul qilish uchun ancha malakali bo'lganlarga ta'sir qilishi mumkin. Me'yor har qanday qaror qabul qilishdan oldin mahsus tajriba va ta'sir doirasi o'lchovini belgilash variantidir.
```

Comment:
`...Scala-dan dasturlash tili sifatida ko'p foydalanardim.` keyingi gapga e'tibor bersangiz: 
`But what's more important about The Standard? ` deb boshlangan. 

`Lekin Me'yor nima uchun muhim?  Me'yor har qanday qaror qabul qilishdan oldin mahsus tajriba, bilim va ta'sir doirasini o'lchash yolidir.` deb davom etting, iltimos.

Reply:
`ilhom rolini o'ynashga mo'ljallangan` o'rniga `ilhomlantirishi ko'zda tutilgan`

Reply:
`etakchilik` => `yetakchilik`

Reply:
👌